src/constants.js

```diff
@@ -1,448 +1,10 @@
-// THIS FILE IS GENERATED, DO NO EDIT MANUALLY
-// For more information see the README.md
-/* eslint-disable dot-notation */
 'use strict'
-module.exports = Object.freeze({
 
-  // serialization
-  PROTOBUF: 0x50,
-  CBOR: 0x51,
-  RLP: 0x60,
-  BENCODE: 0x63,
-  JSON: 0x0200,
-  MESSAGEPACK: 0x0201,
+const table = require('./base-table')
+const constants = {}
```

Comment:
Could you add a comment about the structure of this object, what the keys and values are? This was clear from the original file, but it's hard to get from just reading the code.